frontend/assets/translations/es.json

```diff
@@ -0,0 +1,917 @@
+{
+    "globals": {
+        "readDocumentation": "Leer la documentación",
+        "cancel": "Cancelar",
+        "save": "Guardar",
+        "back": "Atrás",
+        "edit": "Editar",
+        "search": "Buscar",
+        "update": "Actualizar",
+        "delete": "Borrar",
+        "add": "Añadir",
+        "view": "Ver",
+        "create": "Crear",
+        "enabled": "Activado",
+        "disabled": "Desactivado",
+        "yes": "Si",
+        "submit": "Enviar",
+        "select": "Seleccionar",
+        "environmentVar": "Variables de entorno",
+        "saving": "Guardando...",
+        "saveDatasource": "Guardar fuente de datos",
+        "authorize": "Autorizar",
+        "connect": "Conectar",
+        "reconnect": "Reconectar",
+        "components": "componentes",
+        "send": "Enviar",
+        "noConnection": "no se puede conectar",
+        "connectionVerifeid": "conexión verificada",
+        "left": "Izquierda",
+        "center": "Centro",
+        "right": "Derechar",
+        "justified": "Justificado",
+        "host": "Huesped",
```

Comment:
Here, it should have an accent mark on the first `e`, `huésped`. But I don't know if it is worthy to translate 'host' to Spanish, I don't know nowbody that refers to the host as 'huésped' in Spanish.

Reply:
Totally agree on this point, usually i use the english term always, never seen the literal spanish translation for host / hosting stuff, will revert to english this one.
Thanks!